packages/react-dom-bindings/src/events/plugins/FormActionEventPlugin.js

```diff
@@ -53,6 +53,11 @@ function extractEvents(
     if (submitterAction != null) {
       // The submitter overrides the form action.
       action = submitterAction;
+      if (typeof action === 'function') {
```

Comment:
Since there is `if (typeof action !== 'function')` just below… skip this conditional?

Reply:
That's what I ended up doing in the replaying one. Not sure why I didn't see it here.